app/utils/openlayers.ExportControl.js

```diff
@@ -23,17 +24,43 @@ export default class ExportControl extends openlayers.control.Control {
     element.className = `${className || ''} ol-unselectable ol-control`
     element.title = 'Click to export an image of this map'
     element.innerHTML = '<a href="map.png"><i class="fa fa-download"/></a>'
-    element.addEventListener('click', () => this._clicked())
+    const hyperlink = this.element.firstChild
+    hyperlink.addEventListener('click',  this._clicked.bind(this))
   }
 
   _clicked() {
-    const hyperlink = this.element.firstChild
-    const timestamp = new Date().toISOString().replace(/(\D+|\.\d+)/g, '')
     const map = this.getMap()
+    const hyperlink = this.element.firstChild
+    const timestamp = moment().format('llll')
+
     hyperlink.download = `BEACHFRONT_EXPORT_${timestamp}.png`
     map.once('postcompose', event => {
       const canvas = event.context.canvas
-      hyperlink.href = canvas.toDataURL()
+      const imageData = event.context.getImageData(0, 0, canvas.width, canvas.height)
+      const newCanvas = document.querySelector('canvas')
```

Comment:
I think what you want here is to create a new canvas; this just grabs the existing one (which ends up being the main map :) )